code/parse/sexp.cpp

```diff
@@ -3940,7 +3941,7 @@ int check_sexp_syntax(int node, int return_type, int recursive, int *bad_node, i
 					return SEXP_CHECK_TYPE_MISMATCH;
 				}
 
-				if (get_asteroid_position(CTEXT(node)) < 0) {
+				if (stricmp(CTEXT(node), SEXP_NONE_STRING) && (get_asteroid_position(CTEXT(node)) < 0)) {
```

Comment:
This could be submitted on its own as a separate bugfix PR that would not be subject to the code freeze.